en/_posts/2016-04-21-marathon-load-balancer.md

```diff
@@ -6,13 +6,18 @@ author: devatoria
 lang: en
 ---
 
-Marathon load balancer can be configured to use a specific vhost, so the URL to access your application will be more user-friendly. For example, let's say we have a nginx application and we want it to be available at `http://myapp.info`.
+The Marathon load balancer can be configured to use a specific VHost (virtual host), to make you application URL more user-friendly. For example, let's say we have a nginx application and we want it to be available at `https://myapp.info`.
```

Comment:
suggestion: replace vhost notion by "domain name". That's easier to understand without prior LB or Apache knowledge.